python/verticox/datanode.py

```diff
@@ -182,14 +185,17 @@ def _local_update(features: np.array, z: np.array, gamma: np.array, rho,
         return DataNode._compute_sigma(beta, features), beta
 
 
-def serve(features=None, event_times=None, event_happened=None, port=DEFAULT_PORT):
+def serve(features=None, event_times=None, event_happened=None, port=DEFAULT_PORT, timeout=TIMEOUT):
     server = grpc.server(ThreadPoolExecutor(max_workers=1))
```

Comment:
Could `max_workers` in principle be settable to something else? Or will there only ever be one worker at each institution?